.github/workflows/docker-image.yaml

```diff
@@ -0,0 +1,15 @@
+name: Docker Image CI
+
+on:
+  workflow_dispatch:
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build the Docker image
+        run: IMG=openkruise/kruise-manager:${{ github.ref_name }} & make docker-multiarch
```

Comment:
make docker-multiarch will automatically push the image to the repository, this do not want, but you can play the image locally to see if it is successful, but do not push the repository